documentation/sphinx/source/configuration.rst

```diff
@@ -215,8 +213,25 @@ Contains basic configuration parameters of the ``fdbmonitor`` process. ``restart
 
     [general]
     cluster_file = /etc/foundationdb/fdb.cluster
-
-Contains settings applicable to all processes (e.g. fdbserver, backup_agent). The main setting of interest is ``cluster_file``, which specifies the location of the cluster file. This file and the directory that contains it must be writable by all processes (i.e. by the user or group set in the [fdbmonitor] section).
+    restart_delay = 60
+    ## by default, restart_backoff = restart_delay_reset_interval = restart_delay
```

Comment:
It's not actually clear to me from this phrasing if its trying to communicate that setting `restart_delay` to 30 would also change `restart_backoff`, or that this is a note calling out that the defaults are all configured to be equal.

I'd suggest either `Unless configured otherwise, setting restart_delay will also set restart_backoff and restart_delay_reset_interval.` or `_backoff, _reset_interval, and restart_delay all default to 60`

Reply:
The default values of the two parameters are whatever restart delay is, and restart delay is not optional.

Reply:
By the way, the comment comes from the config file, which I didn't intend to change in a patch release

Reply:
Fair enough.

Reply:
I've updated this text for the version of the config in our documentation, and probably I should open a PR to do the same for the actual configuration files. As mentioned, I was thinking not to do the latter in a patch release because changing the config file requires those who use our installers to deal with merging the config files.